src/components/Footer/Footer.component.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import "./Footer.style.css";
+
+const Footer = () => {
+  return (
+    <footer>
+      <div className="div-blocks">
```

Comment:
Avoid naming classes by their elements, so no `div-blocks`, `div-button`. There are times when you can do it, for example `.number-input`